src/io.jl

```diff
@@ -127,6 +129,33 @@ function _loadtable(T, files::Union{AbstractVector,String};
         filegroups = filter(!isempty, map(x->files[x], chunks))
     end
 
+    if blocksize > 0
+        # Use blocked reader
+        blockgroups = []
+        for _files in filegroups
+            for file in _files
+                # Break file into blocks of size `blocksize` or less
+                fsize = filesize(file)
+                nblocks = max(div(fsize, blocksize), 1)
+                bios = blocks(file, '\n', nblocks)
+                # Extract header
+                # FIXME: Use csvread instead!
+                header_str = open(file, "r") do hdrio
+                    readline(hdrio)
+                end
+                header = replace.(string.(split(header_str, ',')),
+                                  Ref(r"\s" => ""))
```

Comment:
This assumes that header is `,` separated. Maybe use TextParse to parse the header?